api.js

```diff
@@ -139,6 +140,64 @@ Api.prototype.observationUpdate = function (req, res, m) {
   })
 }
 
+Api.prototype.observationConvert = function (req, res, m) {
+  var self = this
+
+  res.setHeader('content-type', 'application/json')
+
+  // 1. get the observation
+  this.osm.get(m.id, function (err, obses) {
+    if (err) {
+      res.statusCode = 500
+      res.end('failed to lookup observation: ' + err.toString())
```

Comment:
these aren't returning JSON, but maybe this is a problem throughout the entire server code and could be fixed in a different PR for error handling

Reply:
fixed